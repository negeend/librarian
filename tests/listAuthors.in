LIST AUTHORS
list authors
ADD BOOK books.csv 111111
LIST AUTHORS
add book books.csv 111118
list authors
add book books.csv 111119
LIST AUTHORS
ADD COLLECTION books.csv
LIST AUTHORS
EXIT