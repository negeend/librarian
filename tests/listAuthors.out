user: No books in system.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Harper Lee

user: Successfully added: Harry Potter 1 (J.K. Rowling).

user: Harper Lee
J.K. Rowling

user: Successfully added: Harry Potter 2 (J.K. Rowling).

user: Harper Lee
J.K. Rowling

user: 14 books successfully added.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: Ending Library process.
